src/link-transport.ts

```diff
@@ -19,7 +19,7 @@ export interface LinkTransport {
     /** Called if the request was successful. */
     onSuccess?(request: SigningRequest, result: TransactResult): void
     /** Called if the request failed. */
-    onFailure?(request: SigningRequest, error: Error): void
+    onFailure?(request: SigningRequest | undefined, error: Error): void
```

Comment:
This is a API breaking change, if we want to do this now we should refactor all transport methods to take an identifier

Reply:
I can look at this tomorrow unless you beat me to it. 

I'd like to get a release out tomorrow if possible and start updating all the wallet providers afterwards. You have any other must-have changes before that happens?

Reply:
I went ahead and just reverted this single part of the code - while I still want this as a feature, I don't want it to hold up release.